package.json

```diff
@@ -4,8 +4,8 @@
     "description": "A Social platform for the creative at heart",
     "main": "index.js",
     "scripts": {
-        "pretest": "sequelize db:migrate:undo:all && sequelize db:migrate  && sequelize db:seed:all",
-        "test": "nyc --require @babel/register --require @babel/polyfill mocha src/test/**.spec.js --timeout 20000 --exit",
```

Comment:
why do you have two test script?, I would that you remove one of the test script in the package.json.